spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -1296,6 +1296,19 @@ def register(src, dest, step, param):
             verbose=param.verbose,
         )
 
+    # deep learning registration using cascaded models
+    elif step.algo == 'dl':
+        # check if user provided a mask-- if so, inform it will be ignored
+        if fname_mask:
+            printv('\nWARNING: algo ' + step.algo + ' will ignore the provided mask.\n', 1, 'warning')
+
+        warp_forward_out, warp_inverse_out = register_step_dl_multimodal_cascaded_reg(
```

Comment:
This function is causing another linting issue (GitHub Actions run [here](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/6889986178?check_suite_focus=true#step:5:17)):

```
 spinalcordtoolbox/scripts/sct_register_to_template.py:1305:46: F405 'register_step_dl_multimodal_cascaded_reg' may be undefined, or defined from star imports: spinalcordtoolbox.registration.landmarks, spinalcordtoolbox.registration.register
```

But, this isn't your fault -- This was an existing issue! F405 was triggered because we use a star import here:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/65768163e2e2b00bde46cbf9bc9ee13539fcd5f9/spinalcordtoolbox/scripts/sct_register_to_template.py#L29-L30

(See https://www.flake8rules.com/rules/F405.html for more info.)

For now, you can fix this error by explicitly importing your function:

```diff
 from spinalcordtoolbox.registration.register import * 
+from spinalcordtoolbox.registration.register import register_step_dl_multimodal_cascaded_reg
 from spinalcordtoolbox.registration.landmarks import * 
```

But, outside of this PR, SCT should eventually remove the star-importing in `sct_register_to_template`.

Reply:
> But, outside of this PR, SCT should eventually remove the star-importing in sct_register_to_template

Yes I realized this as well and thought that it was outside the range of this PR. But indeed it's better to start with the correct way of doing it by explicitly importing the register function in this case :) 

So, this is done in this [commit](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3807/commits/f583d1648ea4270b25a5d19dbf1ebcb080c066fe).